tailcfg/tailcfg.go

```diff
@@ -2401,6 +2396,11 @@ const (
 	// NodeAttrSSHEnvironmentVariables enables logic for handling environment variables sent
 	// via SendEnv in the SSH server and applying them to the SSH session.
 	NodeAttrSSHEnvironmentVariables NodeCapability = "ssh-env-vars"
+
+	// NodeAttrServiceHost grants a peer the ability to serve as
+	// a host for a set of given VIP services, which is provided as the
+	// value of this key in NodeCapMap.
```

Comment:
It grants a node the ability, not a peer.

It's probably worth documenting what exactly goes into the value. Perhaps even hold onto this change until you have the struct figured out that you'll use as the value so that you don't need to make another tailcfg change later to add that.

Reply:
I can add that now, since sending down that struct is what I'm working on rn

Reply:
What's the type of the value?

Reply:
The type of the value sent down? That's what Adrian asked me to add, and I'm going to add

It will be of the form:

```
{
  "svc:test": ["100.0.0.1", "fd7a::1"]
}
```

ie. a mapping from service names to their IPv4 addresses.

Reply:
Generally when I ask questions in PRs on documentation, the implication is to make the docs say it, rather than tell one person one time who will forget it. :)

Is there a named type for this type? Let's reference that type name.

Reply:
Oh, sorry, just to be clear - the type doesn't exist yet, it will exist on Monday once I get back to my work laptop 😅 

I can add a link to that in the docs then 👍